backend/initial_setup.docker.py

```diff
@@ -0,0 +1,229 @@
+import logging
+import os
+import subprocess
+from typing import Dict, List, Tuple
+
+from flask import Flask, g, request, redirect, send_from_directory
+from flask import session as flask_session
+from flask_compress import Compress
```

Comment:
same here with complete duplication

Reply:
same as before, don't want to disrupt current development environments